tensorflow/python/keras/backend.py

```diff
@@ -1212,24 +1212,30 @@ def zeros(shape, dtype=None, name=None):
   """Instantiates an all-zeros variable and returns it.
 
   Arguments:
-      shape: Tuple of integers, shape of returned Keras variable
-      dtype: String, data type of returned Keras variable
-      name: String, name of returned Keras variable
+      shape: Tuple or list of integers, shape of returned Keras variable
+      dtype: data type of returned Keras variable
+      name : name of returned Keras variable
```

Comment:
I don't think pylint will approve of the extra space between `name` and `:`

Reply:
Thanks @mihaimaruseac. I have updated it.